### ISSUE TITLE
[DONATION] Counter Movement Jump

### ISSUE THREAD

**Issue by lnthach** [uea-machine-learning/tsml_repo#10]
**Please give a brief description of the data**

Accelerometer data of ten participants performing Counter Movement Jump test.

**Where can we get the data?**
provide a link or ask us to contact you directly if that is better
https://github.com/lnthach/Mr-SEQL/tree/master/data/CMJ

**Is there a publication related to the data**

The paper: http://link.springer.com/article/10.1007/s10618-019-00633-3
The data is part of a case-study to demonstrate the accuracy and interpretability of our proposed time series classifier. Our classifier extracted SAX and/or SFA sequences to be used as features for model training. We also experimented some other state-of-the-art methods with this data. The results can be found on our GitHub https://github.com/lnthach/Mr-SEQL/.

**Is your data multivariate or univariate?**
Both. The raw multivariate time series have three components (x,y, and z) of the accelerometer signals. The univariate data is a series of acceleration magnitude, i.e., sqrt(x^2 + y^2 + z^2).
Univariate: https://github.com/lnthach/Mr-SEQL/tree/master/data/CMJ/Jump
Multivariate: https://github.com/lnthach/Mr-SEQL/tree/master/data/CMJ/JumpMV

**Is there a default train/test split for the data**
If not, we will create one randomly. If this is inappropriate (e.g. want to avoid test subjects being in both train and test splits) let us know and advise on how we should split

We splitted the data based on the participants. Out of ten participants, seven were put in the training set and three were put in the test set. 

**Are there any missing values/padding?**
We do not want to pre pad data. If the series are not equal length, please let us know

There are no missing or padding values. The time series are not equal. However in our publication we also experimented with pre-processed data by cropping and resampling. Therefore, we provide in our repository two other sets: JumpCropped and JumpResampled. In the former, we cropped the uneventful beginning and ending parts of the time series (the results are still unequal length). In the latter, all cropped time series were resampled to equal length of 500. 

**How would you like the donation attributed on the website?**

Data created by Martin O'Reilly. Data donated by Thach Le Nguyen, Martin O'Reilly, and Georgiana Ifrim.

**Can you provide a paragraph description, including the meaning of the class values, for the website?**

Ten participants were recruited to perform the Counter Movement Jump (CMJ) test. The participants were asked to complete the test with acceptable form, with their legs bending during flight, and with a stumble upon landing. Participants were equipped with a Shimmer 3 inertial measurement unit (IMU) on their dominant foot. The IMU was configured to stream wide range, tri-axial accelerometer data at 1024Hz. The result is a 3-class (of three different forms) time series dataset: 0 means "acceptable form", 1 means "legs bending during flight", and 2 means "stumble upon landing".   

If you have further questions, please let me know.

Best,
Thach

**Comment by TonyBagnall** [comment]
hi, sorry for the delay, I've cloned it and will take a look.

**Comment by TonyBagnall** [comment]
Finally we have done this! Just the multivariate version for now
